features/admin.users.v1/components/user-profile.tsx

```diff
@@ -222,9 +256,29 @@ export const UserProfile: FunctionComponent<UserProfilePropsInterface> = (
                         ...configurationStatuses,
                         accountLock: property.value
                     };
+                } else if (property.name === ServerConfigurationsConstants.ENABLE_MULTIPLE_EMAILS_AND_MOBILE_NUMBERS) {
+                    configurationStatuses = {
+                        ...configurationStatuses,
+                        isMultipleEmailAndMobileNumberEnabled: property.value
+                    };
+                } else if (property.name === ServerConfigurationsConstants.ENABLE_EMAIL_VERIFICATION) {
+                    configurationStatuses = {
+                        ...configurationStatuses,
+                        isEmailVerificationEnabled: property.value
+                    };
+                } else if (property.name === ServerConfigurationsConstants.ENABLE_MOBILE_VERIFICATION) {
+                    configurationStatuses = {
+                        ...configurationStatuses,
+                        isMobileVerificationEnabled: property.value
+                    };
+                } else if (property.name
+                    === ServerConfigurationsConstants.ENABLE_MOBILE_VERIFICATION_BY_PRIVILEGED_USER) {
+                    configurationStatuses = {
+                        ...configurationStatuses,
+                        isMobileVerificationByPrivilegeUserEnabled: property.value
+                    };
```

Comment:
using the switch statement will be more cleaner. Refactor this based on the assigned time.

Reply:
Changed